.github/workflows/test.yml

```diff
@@ -83,6 +83,14 @@ jobs:
         export ARTIFACT_GOOGLE_APPLICATION_CREDENTIALS=`pwd`/artifact_google_creds.json
         sudo -u testbot --preserve-env=ARTIFACT_GOOGLE_APPLICATION_CREDENTIALS,TEST_MONGODB_URI,MONGODB_TEST_OUTPUT_URI,GITHUB_SHA,GITHUB_RUN_ID,GITHUB_RUN_NUMBER,GITHUB_RUN_ATTEMPT,GITHUB_X_PR_BASE_SHA,GITHUB_X_PR_BASE_REF,GITHUB_X_HEAD_REF,GITHUB_X_HEAD_SHA,GITHUB_REPOSITORY bash -lc 'make cover'
 
+    - name: Upload test.json
+      if: always()
+      uses: actions/upload-artifact@v3
+      with:
+        name: test.json
+        path: json.log
+        retention-days: 30
+
     # this one runs as root for CAP_SETUID
```

Comment:
Not sure what to do about this. For now I'm fine if we just omit these results. Unless it becomes particularly noisy.

As far as ideas -- can I assume the same machine is running this? And that `json.log` already exists? Can I just change `run` to append to `json.log`. E.g:
```
    - name: test run-as-user
      env:
        TEST_SUBPROC_USER: testbot
      run: go test -v ./pexec -run TestManagedProcessStart >> json.log
```

Reply:
that cap_setuid test is mine -- fine to omit, fine to `>>` append, your call

(steps in the same job run in the same machine)